app/screens/edit_post/post_error/index.tsx

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {StyleSheet, useWindowDimensions, View} from 'react-native';
+
+import ErrorText from '@components/error_text';
+import {useTheme} from '@context/theme';
+
+type PostErrorProps = {
+    errorLine?: string;
+    errorExtra?: string;
+}
+
+const styles = StyleSheet.create({
+    errorContainerSplit: {
+        paddingHorizontal: 15,
+        flexDirection: 'row',
+        justifyContent: 'space-between',
+    },
+    errorContainer: {
+        paddingHorizontal: 10,
+    },
+    errorWrap: {
+        flexShrink: 1,
+        paddingRight: 20,
+    },
+    errorWrapper: {
+        alignItems: 'center',
+    },
+});
+
+const PostError = ({errorLine, errorExtra}: PostErrorProps) => {
+    const theme = useTheme();
```

Comment:
perhaps we should move the theme to the `ErrorText` component

Reply:
Done.